pandas/core/arrays/numeric.py

```diff
@@ -199,3 +201,48 @@ def reconstruct(x):
             return tuple(reconstruct(x) for x in result)
         else:
             return reconstruct(result)
+
+    def _apply(self, func: Callable, **kwargs) -> NumericArray:
+        """
+        Applying a given function on the array
+
+        Parameters
+        ----------
+        func : callable function to apply
+        **kwargs
+            additional arguments for the function if necessary
+
+        Returns
+        -------
+        NumericArray
+            The image of the NumericArray
+        """
+        values = func(self._data, **kwargs)
+        return type(self)(values, self._mask.copy())
+
+    def round(self, decimals: int = 0, *args, **kwargs) -> NumericArray:
+        """
+        Round each value in NumericArray a to the given number of decimals.
```

Comment:
```suggestion
        Round each value in the array to the given number of decimals.
```

(NumericArray is not really public, so would try to avoid it in the docstring where it's not really needed)